app/Models/OutcomeData.php

```diff
@@ -34,7 +34,7 @@ public static function isSubmitted($actor_id, $activity_id, $submission_id)
      */
     public static function getUniqueChapters($actor_id, $activity_id, $submission_id) 
     {
-        $result = DB::select("select distinct chapter_name , page_order from outcome_data where user_id = ? AND submission_id = ?  AND assignment_id = ?  AND verb IN(?,?) order by page_order", [$actor_id, $submission_id, $activity_id,'interacted','answered']);
+        $result = DB::select("select distinct chapter_name , page_order from outcome_data where user_id = ? AND submission_id = ?  AND assignment_id = ?  AND verb IN(?, ?)  order by page_order", [$actor_id, $submission_id, $activity_id,'interacted','answered']);
```

Comment:
Please add space after comma 
i.e. "$activity_id,'interacted','answered'"
should be like "$activity_id, 'interacted', 'answered'"